cdap-ui/app/cdap/components/PluginJSONCreator/Create/Content/JsonMenu/index.tsx

```diff
@@ -154,6 +169,10 @@ const JsonMenuView: React.FC<WithStyles<typeof styles>> = ({ classes }) => {
     setWidgetInfo(newWidgetData.widgetInfo);
     setWidgetToAttributes(newWidgetData.widgetToAttributes);
     setOutputName(newWidgetData.outputName);
```

Comment:
Question, should we also make `outputLabel`?

Reply:
no, I don't think we're showing output label